package.json

```diff
@@ -4,37 +4,35 @@
   "description": "Application to connect Hemi Network to External Services",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "type": "module",
   "private": true,
   "scripts": {
-    "give-points": "bun ./src/presentation/cli/GivePoints.ts",
     "lint": "eslint",
     "lint:fix": "eslint --fix",
     "test": "vitest run",
     "test:cov": "vitest run --coverage",
-    "test:watch": "vitest watch"
+    "test:watch": "vitest watch",
+    "build": "tsc -p ./tsconfig.build.json",
+    "collect-metrics": "node dist/presentation/cli/CollectMetrics.js",
+    "give-points": "node ./src/presentation/cli/GivePoints.ts"
```

Comment:
I recommend sorting all these commands alphabetically so it's easier to find as this list grows. You can even go further and use our [better-sort-package-json](https://www.npmjs.com/package/better-sort-package-json) package that, once configured with `lint-staged`, does this automatically for you 😄